### PR TITLE
typo: Double words

### PR DESCRIPTION
**Pull request by nschonni**
Ran Vale across the repo to find them and did one commit per word rather than multiple PRs by spec.
Looks like there might be some encoding issue in `css-page-3/lc-doc-20040215.htm` that's causing the large line diff count

### CODE REVIEW

File: css-device-adapt-1/Overview.bs [hunk 1]
```diff
@@ -888,7 +888,7 @@ query affected by the viewport descriptors.
 
 It is recommended that authors do not write ''@viewport'' rules that
 rely on media queries whose evaluation is affected by viewport
-descriptors. Is is also recommended that the ''@viewport'' rule(s) is
```
Comment (tabatkins) [inline css-device-adapt-1/Overview.bs:891]: This isn't a repetition, it's meant to be "It is"

File: css-images-3/Overview.bs [hunk 2]
```diff
@@ -116,7 +116,7 @@ Image Values: the <<image>> type {#image-values}
 	allowing the 'list-style-type' to render in its place. [[CSS2]]
 
 	While an image is loading,
-	is is a <dfn export>loading image</dfn>.
```
Comment (tabatkins) [inline css-images-3/Overview.bs:119]: another "it is" intended

File: css-images-4/Overview.bs [hunk 3]
```diff
@@ -89,7 +89,7 @@ Value Definitions {#values}
 	allowing the 'list-style-type' to render in its place. [[CSS2]]
 
 	While an image is loading,
-	is is a <dfn export>loading image</dfn>.
```
Comment (tabatkins) [inline css-images-4/Overview.bs:92]: "it is"

File: css-page-3/lc-doc-20040215.htm [hunk 4]
```diff
@@ -1,9377 +1,9407 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```
Comment (tabatkins) [inline css-page-3/lc-doc-20040215.htm:1]: Woah, what happened with this file?

File: css-speech-1/Overview.bs [hunk 5]
```diff
@@ -532,7 +532,7 @@ The 'speak-as' property</h3>
 			Spells the text one letter at a time (useful for acronyms and abbreviations).
 			In languages where accented characters are rare,
 			it is permitted to drop accents in favor of alternative unaccented spellings.
-			As as example, in English, the word “r&ocirc;le” can also be written as “role”.
```
Comment (tabatkins) [inline css-speech-1/Overview.bs:535]: Should be "as an"

File: css-tables-3/Overview.bs [hunk 6]
```diff
@@ -951,7 +951,7 @@ spec:css-sizing-3; type:property; text:box-sizing
 				<li>The 'border-spacing' of the <a>table-root</a> is ignored (as if it was set to 0px).
 
 				<li>The 'border-radius' of both <a>table-root</a> and <a>table-non-root</a> boxes
-					is ignored (as it it was set to 0px).
```
Comment (tabatkins) [inline css-tables-3/Overview.bs:954]: should be "if it"

File: css-text-3/test-coverage.html [hunk 7]
```diff
@@ -43,7 +43,7 @@ <h2 class="no-num no-toc no-ref heading settled" id="abstract"><span class="cont
 	and each section links to the corresponding one in he specification.
 
 <div class=issue>Issue:
-	<p>Problems other than missing spec coverage are highlighted he same as as this paragraph is.
```
Comment (tabatkins) [inline css-text-3/test-coverage.html:46]: correct, but also "he"->"the" on this line

File: css2/Overview.bs [hunk 8]
```diff
@@ -3277,7 +3277,7 @@ div * p
 DIV element. Note the white space on either side of the "*" is not part
 of the universal selector; the white space is a combinator
 indicating
-that the DIV must be the ancestor of some element, and that that
```
Comment (tabatkins) [inline css2/Overview.bs:3280]: This is intentional.

File: selectors-3/Overview.html [hunk 9]
```diff
@@ -2850,7 +2850,7 @@ <h3 id=descendant-combinators><span class=secno>8.1. </span>Descendant
     descendant of a <code>div</code> element. Note the whitespace on either
     side of the "*" is not part of the universal selector; the whitespace is
     a combinator indicating that the <code>div</code> must be the ancestor of
-    some element, and that that element must be an ancestor of the
```
Comment (tabatkins) [inline selectors-3/Overview.html:2853]: This is intentional.

File: selectors-3/Overview.src.html [hunk 10]
```diff
@@ -2218,7 +2218,7 @@ <h3 id=descendant-combinators>Descendant combinator</h3>
  descendant of a <code>div</code> element. Note the whitespace on
  either side of the "*" is not part of the universal selector; the
  whitespace is a combinator indicating that the <code>div</code> must be the
- ancestor of some element, and that that element must be an ancestor
```
Comment (tabatkins) [inline selectors-3/Overview.src.html:2221]: This is intentional.

File: selectors-4/Overview.bs [hunk 11]
```diff
@@ -3096,7 +3096,7 @@ Descendant combinator (<code> </code>)</h3>
 		descendant of a <a element>div</a> element. Note the whitespace on
 		either side of the "*" is not part of the universal selector; the
 		whitespace is a combinator indicating that the <code>div</code> must be the
-		ancestor of some element, and that that element must be an ancestor
```
Comment (tabatkins) [inline selectors-4/Overview.bs:3099]: Intentional.

**Review by tabatkins — CHANGES_REQUESTED** [review]
A few necessary fixes, otherwise lgtm

File: css-page-3/lc-doc-20040215.htm (hunk 4 above)
Comment (nschonni) [reply css-page-3/lc-doc-20040215.htm:1]: Not sure if there was some encoding issue, but I can always remove it given the age

**Comment by nschonni** [comment]
@tabatkins dropped the "that" commit, and address the other comments except for the file that got "reencoded" or something. Don't think it was a line ending thing, since those were handled in another PR

File: css-page-3/lc-doc-20040215.htm (hunk 4 above)
Comment (tabatkins) [reply css-page-3/lc-doc-20040215.htm:1]: yeah, we should just, uh, not touch that file i guess.  It's a generated file anyway.

Comment (nschonni) [reply css-page-3/lc-doc-20040215.htm]: Rebased it out

**Pull request merged**